app/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mk.edu.uklo.fikt.fiktexam" >
+    package="mk.edu.uklo.fikt.fiktexam">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    
```

Comment:
Please use the standard  formatter CRTL + ALT + L (this might vary from pc to pc)